pkgs/by-name/xo/xow_dongle-firmware/package.nix

```diff
@@ -4,36 +4,48 @@
   fetchurl,
   cabextract,
 }:
-
 stdenvNoCC.mkDerivation rec {
   pname = "xow_dongle-firmware";
-  version = "2017-07";
+  version = "0-unstable-2025-04-22";
 
-  dontConfigure = true;
-  dontBuild = true;
-
-  src = fetchurl {
+  xow_dongle = fetchurl {
```

Comment:
I think this change might trigger IFD (import-from-derivation). I guess the right thing would be to use `srcs`: https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-src